docs/ARDUPILOT.md

```diff
@@ -0,0 +1,55 @@
+# mLRS Documentation: ArduPilot Setup #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
+
+Further parameters can be configured to optimize peformance for ArduPilot and the RF mode (19Hz, 31Hz or 50Hz) selected.
+
+## mLRS Rx Module Setup
+
+- Rx Snd RadioStat= ardu_1
+
+## Serial Port Setup
+
+- SERIALx_BAUD:
+    - 57 for 31Hz, 50Hz
+    - 38 for 19Hz
+- SERIALx_OPTIONS = 4096 (ignore commands from GCS to change stream rates)
+
+Note: These baud rates enable MAVFTP parameter download
+
+## Stream Rates Setup
+
+- SRx_ADSB = 0
+- SRx_EXT_STAT:
+    - 2 for 31Hz, 50Hz
+    - 1 for 19Hz
+- SRx_EXTRA1 = 4
+- SRx_EXTRA2 = 4
+- SRx_EXTRA3:
+    - 2 for 31Hz, 50Hz
+    - 1 for 19Hz
+- SRx_PARAMS = 50
+- SRx_POSITION = 2
+- SRx_RAW_CTRL = 0
+- SRx_RAW_SENS = 0 (for most of you this one is unimportant, keep it at 0 unless you really need it)
+- SRx_RC_CHAN = 0
+
+## CRSF Receiver Setup
+
+Setting up ArduPilot for a CRSF receiver can be a bit tricky, as it depends on the flight controller board, and might need BRD_ALT_CONFIG to be set to a specific value. It is best to consult the ArduPilot wiki, or ask in the ArduPilot discussion channel.
```

Comment:
this piece IMHO should be in CRSF.

Reply:
Added CRSF receiver section under Ardupilot Setup on CRSF page.